debian/stretch/foreman/foreman-assets.postinst

```diff
@@ -29,6 +29,9 @@ fi
 # Own all the core files
 chown -Rf foreman:foreman '/usr/share/foreman'
 
+# make webpack file executable
+chmod 755 '/usr/share/foreman/node_modules/webpack/bin/webpack.js'
```

Comment:
Shouldn't it live under `./node_modules/.bin/webpack`?

Reply:
That's a symlink to the actiual file - at least in the environment, this got built on.